great_expectations/data_context/templates.py

```diff
@@ -11,6 +11,8 @@
 #   - Read our docs: https://docs.greatexpectations.io/en/latest/reference/data_context_reference.html#configuration
 #   - Join our slack channel: http://greatexpectations.io/slack
 
+# config_version tells Great Expectations whether the config file has been generated by GE version 0.8.0 or later.
+# note: This usually does not need to be changed.
```

Comment:
@spbail If you have any suggestions on how this note can be changed please let me know

Reply:
Hm I think the fact that it's generated by 0.8.0 or later isn't really relevant and might just be confusing? It's more like "config_version represents the syntax version of this config file. It's auto-generated and usually does not need to be changed". Not sure if the term "syntax version" makes any sense haha.

Reply:
agree that "version 0.8.0 ..etc etc" may be too much.. I was definitely on the fence as I was writing it :) Thanks for your suggestions!

Reply:
how about this version?